packages/back-end/src/api/openapi/payload-schemas/PostExperimentPayload.yaml

```diff
@@ -201,6 +210,12 @@ properties:
   regressionAdjustmentEnabled:
     type: boolean
     description: Controls whether regression adjustment (CUPED) is enabled for experiment analyses
+  sequentialTestingEnabled:
```

Comment:
I wish these were all in "settings" just like the return payload :/ 

not worth changing now...